.travis.yml

```diff
@@ -8,22 +8,9 @@ python:
   - 3.5
 
 env:
-  - TOX_ENV=django3
-  - TOX_ENV=django22
-  - TOX_ENV=django21
```

Comment:
We want to continue testing at least with Django 2.1 and 2.2, since we'll be upgrading through those soon.